build.gradle

```diff
@@ -20,7 +20,7 @@ publishing {
 }
 
 group = 'com.github.hmcts'
-version = '1.4.2'
+version = '1.4.3'
```

Comment:
This will now need updating to 1.4.4 as there's a new version that's gone in.

Reply:
Version upgraded